account_invoice_search_by_reference/tests/test_account_invoice_search_by_reference.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountInvoiceSearchByReference(TransactionCase):
+    """
+        Tests for Account Invoice Merge.
```

Comment:
you forgot to remove this

Reply:
@lreficent  Removed line, Could you please review it.